GCRCatalogs/dc2_dia_object.py

```diff
@@ -70,26 +70,26 @@ def _generate_modifiers(dm_schema_version=4, bands='ugrizy', **kwargs):  # pylin
             'diaObjectId': 'diaObjectId',
             'ra': (np.rad2deg, 'coord_ra'),
             'dec': (np.rad2deg, 'coord_dec'),
-#            'radecCov':  # A covariance matrix for the uncertainty in ra, dec
-#            'radecTai': 'dateobs'?  # I don't know what this is called
-#            'pm':
-#            'pmParallaxCov':
-#            'pmParallaxLnl':
-#            'pmParallaxChi2':
-#            'pmParallaxNdata':
-#            'totFluxMean': 'ip_diffim_forced_PsfFlux_instFlux',
-#            'totFluxMeanErr': 'ip_diffim_forced_PsfFlux_instFluxErr',
-#            'totFluxSigma': 'ip_diffim_forced_PsfFlux_instFluxErr',
-#            'lcPeriodic'
-#            'lcNonPeriodic'
-#  These are possible to calculate if you require the Object Table
-#            'nearbyObj':
-#            'nearbyObjDist':
-#            'nearbyObjLnP':
+            # 'radecCov': '',  # A covariance matrix for the uncertainty in ra, dec
+            # 'radecTai': 'dateobs',  # I don't know what this is called
+            # 'pm': '',
+            # 'pmParallaxCov': '',
+            # 'pmParallaxLnl': '',
+            # 'pmParallaxChi2': '',
+            # 'pmParallaxNdata': '',
+            # 'totFluxMean': 'ip_diffim_forced_PsfFlux_instFlux',
+            # 'totFluxMeanErr': 'ip_diffim_forced_PsfFlux_instFluxErr',
+            # 'totFluxSigma': 'ip_diffim_forced_PsfFlux_instFluxErr',
+            # 'lcPeriodic': '',
+            # 'lcNonPeriodic': '',
+            # # These are possible to calculate if you require the Object Table
+            # 'nearbyObj': '',
+            # 'nearbyObjDist': '',
+            # 'nearbyObjLnP': '',
         }
 
-        modifiers['good'] = (create_basic_flag_mask,)
-        modifiers['clean'] = modifiers['good']
+        # modifiers['good'] = (create_basic_flag_mask, )
```

Comment:
Why are these now commented out?  Should they have not been there in the first place?

Reply:
This line appears to be an unfinished line. There should be columns names following the `create_basic_flag_mask` function ([example](https://github.com/LSSTDESC/gcr-catalogs/blob/master/GCRCatalogs/dc2_object.py#L360)). 

However, I checked and could not find these columns needed to define `good` in the underlying file, so I commented out them for now. We'll need to check with @wmwv if we want to properly resolve this.

Reply:
Good enough for now.  Thanks.